ibtk/include/ibtk/HierarchyMathOps.h

```diff
@@ -498,6 +498,37 @@ class HierarchyMathOps : public SAMRAI::tbox::DescribedClass
              int dst_depth = 0,
              int src2_depth = 0);
 
+    /*!
+     * \brief Compute the cell-centered divergence of product of two side-centered normal
+     * vectors field using centered differences.
+     *
+     * Sets dst = alpha div (coef1 * src1) + beta src2.
```

Comment:
I'm not sure this is a meaningful distinction, but where this is used in practice, is `coef1` typically describing coefficients that "hit" `src1`, or is it a solution variable itself --- e.g. `alpha div (u v) + beta w` vs. `alpha div (beta u) + gamma w`?